Docs/Configuration.tex

```diff
@@ -6293,20 +6328,20 @@ \subsection{Output Properties}\label{uefioutputprops}
   The use of \texttt{BuiltinGraphics} is generally straightforward.
   For most platforms it is necessary to enable \texttt{ProvideConsoleGop},
   set \texttt{Resolution} to \texttt{Max}. \texttt{BuiltinText} variant is
-  an alternative \texttt{BuiltinGraphics} for some very old and faulty
+  an alternative \texttt{BuiltinGraphics} for some very old and defective
   laptop firmware, which can only draw in \texttt{Text} mode.
 
-  The use of \texttt{System} protocols is more complicated. In general
+  The use of \texttt{System} protocols is more complicated. Typically,
   the preferred setting is \texttt{SystemGraphics} or \texttt{SystemText}.
   Enabling \texttt{ProvideConsoleGop}, setting \texttt{Resolution} to
   \texttt{Max}, enabling \texttt{ReplaceTabWithSpace} is useful on almost
   all platforms. \texttt{SanitiseClearScreen}, \texttt{IgnoreTextInGraphics},
   and \texttt{ClearScreenOnModeSwitch} are more specific, and their use
   depends on the firmware.
 
-  \emph{Note}: Some Macs, namely \texttt{MacPro5,1}, may have broken
-  console output with newer GPUs, and thus only \texttt{BuiltinGraphics}
-  may work for them.
+  \emph{Note}: Some Macs, such as the \texttt{MacPro5,1}, may have incompatible
+  console output when using modern GPUs, and thus only \texttt{BuiltinGraphics}
+  may work for them in such cases.
```

Comment:
It is worth adding that NVIDIA GPUs need additional firmware upgrades and linking  to https://github.com/acidanthera/bugtracker/issues/1280.